src/Markdig/Extensions/DefinitionLists/DefinitionListParser.cs

```diff
@@ -103,6 +102,19 @@ public override BlockState TryOpen(BlockProcessor processor)
             return BlockState.Continue;
         }
 
+        private static DefinitionList GetCurrentDefinitionList(ParagraphBlock paragraphBlock, ContainerBlock previousParent)
+        {
+            var index = previousParent.IndexOf(paragraphBlock) - 1;
+            if (index < 0) return null;
+            var lastBlock = previousParent[index];
+            if (lastBlock is BlankLineBlock)
```

Comment:
I could use a while loop to traverse back until it finds anything that _isn't_ a `BlankLineBlock` and returns that as `DefinitionList`, but how would the parent container handle these extra lines? Should they be removed, or could the container handle them?

```c#
private static DefinitionList GetCurrentDefinitionList(ParagraphBlock paragraphBlock, ContainerBlock previousParent)
{
    var index = previousParent.IndexOf(paragraphBlock) - 1;
    if (index < 0) return null;
    var lastBlock = previousParent[index];
    while (lastBlock is BlankLineBlock && index > 0)
    {
        lastBlock = previousParent[--index];
    }
    return lastBlock as DefinitionList;
}
```